native/core/src/execution/serde.rs

```diff
@@ -111,7 +111,7 @@ pub fn to_arrow_datatype(dt_value: &DataType) -> ArrowDataType {
         {
             DatatypeStruct::List(info) => {
                 let field = Field::new(
-                    "item",
+                    "element",
```

Comment:
This change was necessary to resolve an error trying to cast `List(Field { name: "element", ..` to `List(Field { name: "item", ..` in schema_adapter